### PR TITLE
DnD Monsters - React Capstone Project

### PR DESCRIPTION
**Pull request by Arch-Noize**
This project is a mobile, single page application that displays monsters from the popular TTRPG Dungeons and Dragons, 5th Edition. Using [Open5eAPI](https://open5e.com/api-docs), React, React Bootstrap, and Redux.

This website currently displays a list of monsters ordered in alphabetical order, then by their danger level. You can also:
- Find more information of each monster when clicking the card
- Filter the list of monsters choosing books from a dropdown

### CODE REVIEW

File: src/styles/Monster.module.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  width: 600px;
+  max-width: 500%;
+  margin: 5px 0 5px -145px;
+}
+
+.card button {
+  margin-left: 100px;
+}
+
+.doc {
```
Comment (rloterh) [inline src/styles/Monster.module.css]: - [ ] Good job, card details are displaying correctly on a mobile screen. However, let's ensure that the styling of the list on the _details page_ conforms to the striped color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that the card details follow the **alternating striped color pattern**. First, deep version of the color, and then, lighter a lighter version of the same color, and it continues in that fashion.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
Home Page | Home Page

File: README.md [hunk 2]
```diff
@@ -46,15 +46,17 @@ Website currently uses:
 <!-- Features -->
```
Comment (rloterh) [inline README.md:47]: - [ ] Nice and smart acknowledging Nelson Sakwa, the creator of the original design in the ReadMe, however, let's ensure that this falls under the _Acknowledgement_ section of the ReadMe, around _line 161_.

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Salt, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there!  
🏁  🌴 🚦 🚗 🏫  🏡   _🏃 

## Highlights 👍  
- Nice job on presenting your project!
- SPA feature working as intended!
- Properly implemented state management using Redux!
- Filter properly implemented!
- Good job with the tests!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
Let's make the ReadMe stand out even better by including instructions under the _Getting Started_ section, on how to run test cases for the app, for instance, using the command, `npm test`.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @Arch-Noize, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Add credit in `acknowledgment` ✅ 
- Change the details page design to match the provided design ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**